eng/Versions.props

```diff
@@ -15,7 +15,7 @@
   </PropertyGroup>
   <!-- Versioning for assemblies/packages -->
   <PropertyGroup>
-    <MajorVersion>7</MajorVersion>
+    <MajorVersion>8</MajorVersion>
```

Comment:
Think we need to keep this at 7 for now. This controls the assembly version that we stamp into our own binaries. That should not change until we actually ship .NET 8.